Blockstack/Classes/EncryptionJS.swift

```diff
@@ -47,9 +47,6 @@ open class EncryptionJS {
         return context
     }()
     
-    init() {
-    }
-    
     public func decryptECIES(privateKey: String, cipherObjectJSONString: String) -> String? {
```

Comment:
blockstack.js's `decryptECIES` will return a `Buffer` when the encrypted content isn't string.